src/js/tracker.js

```diff
@@ -1472,8 +1472,18 @@
 				helpers.addEventListener(documentAlias, 'mouseup', activityHandler);
 				helpers.addEventListener(documentAlias, 'mousedown', activityHandler);
 				helpers.addEventListener(documentAlias, 'mousemove', activityHandler);
-				helpers.addEventListener(documentAlias, 'mousewheel', activityHandler);
-				helpers.addEventListener(windowAlias, 'DOMMouseScroll', activityHandler);
+				// https://github.com/Modernizr/Modernizr/blob/a9369b031d05b90231b572e2796afba0334b21e6/feature-detects/dom/passiveeventlisteners.js#L26
+				var supportsPassiveOption = false;
+				try {
+					var opts = Object.defineProperty({}, 'passive', {
+						get: function() {
+							supportsPassiveOption = true;
+						}
+					});
+					window.addEventListener('test', null, opts);
+				} catch (e) {}
+				helpers.addEventListener(documentAlias, 'mousewheel', activityHandler, supportsPassiveOption ? { passive: true } : undefined);
+				helpers.addEventListener(windowAlias, 'DOMMouseScroll', activityHandler, supportsPassiveOption ? { passive: true } : undefined);
```

Comment:
Actually I'm not sure we need `DOMMouseScroll`, as it's already async apparently:

> The DOM DOMMouseScroll event is fired asynchronously when mouse wheel

https://developer.mozilla.org/en-US/docs/Web/Events/DOMMouseScroll

Reply:
yup I think we could remove scroll events but wdyt about adding touch events?

Reply:
As far as I can see, Snowplow isn't adding blocking event listeners for any of the touch events, so we have to need to make them passive.

I'll remove `DOMMouseScroll`.

Reply:
Updated!